recipe/meta.yaml

```diff
@@ -76,12 +75,11 @@ outputs:
       script: cd md && {{ PYTHON }} -m pip install . --no-deps -vv
     requirements:
       host:
-        - python >=2.7
+        - python >=3
```

Comment:
i'd probably go ahead and make this `>=3.7` at this point, as there are probably no `3.6`-compatible upstream versions forthcoming...

Reply:
Makes sense. Thanks for the tip, I hadn't checked the upstream versions yet :).

Reply:
Noticed:

> Python version: Python 3.6+ (only 64 bit)

on the [spaCy readme](https://github.com/explosion/spaCy#-install-spacy), so setting it to >= py36 instead.